src/calculator.js

```diff
@@ -0,0 +1,32 @@
+import './calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <div className="display"><span>0</span></div>
+      <div className="buttons">
+        <button type="button">AC</button>
+        <button type="button">+/-</button>
```

Comment:
- [ ] According to the project requirements, You should create a directory called `components` which should contain a `Calculator.js` file. Kindly create this and move the `Calculator.js` file inside the components directory. 👍🏻